PovioKit/Classes/Utilities/Throttler/README.md

```diff
@@ -0,0 +1,23 @@
+#  Throttler
+
+Job manager executing only latest scheduled job after set delay.
+
+## Example
+
+We used `Throttler` for search functionality, where we delayed requesting search results. Executing a request each key press would be quite an overhead, and the throttler comes to the rescue.
```

Comment:
We are not bound to search functionality here. Can you please re-phrase this sentence to be more generic?

Reply:
This is just an example of usage.

Reply:
Ah, got it. But you could still re-write the sentence so you are actually writing to the reader e.g. `You can use Throttler ...`

Reply:
I agree, will fix.